packages/block-library/src/cover/shared.js

```diff
@@ -200,3 +203,30 @@ export async function getCoverIsDark( url, dimRatio = 50, overlayColor ) {
 	const composite = compositeSourceOver( overlay, background );
 	return colord( composite ).isDark();
 }
+
+export async function getAverageMediaColor( url ) {
+	if ( url ) {
+		try {
+			const imgCrossOrigin = applyFilters(
+				'media.crossOrigin',
+				undefined,
+				url
+			);
+			const color = await retrieveFastAverageColor().getColorAsync( url, {
+				// Previously the default color was white, but that changed
+				// in v6.0.0 so it has to be manually set now.
+				defaultColor: [ 255, 255, 255, 255 ],
+				// Errors that come up don't reject the promise, so error
+				// logging has to be silenced with this option.
+				silent: process.env.NODE_ENV === 'production',
+				crossOrigin: imgCrossOrigin,
+			} );
+			// FAC uses 0-255 for alpha, but colord expects 0-1.
+			return color.hex;
+		} catch ( error ) {
+			// If there's an error, just assume the image is dark.
+			return false;
```

Comment:
In https://github.com/WordPress/gutenberg/pull/54054/commits/dc9d046ffdbe56a93525ffee2b5d6937ab66e8e9 I've set the default to be #000 and return undefined if there is an error. Not sure how to approach otherwise?